Fix files for release (#81)
nf-core/magmap: docs/usage.md

The indexes input is used by Sourmash to select genomes that can be downloaded in a second step and added to the pipeline.
It is provided with the `--indexes` parameter and it is a path (local or remote).

```bash
nextflow run nf-core/magmap -profile docker --outdir results/ --input samples.csv --genomeinfo localgenomes.csv --indexes 'https://github.com/nf-core/test-datasets/raw/magmap/testdata/sourmash_test.index.sbt.zip'
```

N.B.: the sbt files can be generated with sourmash (check it [here](https://sourmash.readthedocs.io/en/latest/index.html)) or some can be found in sourmash documentation website.
E.g. "sourmash gtdb sbt" you will find the indexes for gtdb.

#### Metadata input

> Magmap accepts several metadata as `.csv` files that provides information about the genomes that you will use in the pipeline. Each metadata can be used to get different information about you genomes as their quality (CheckM) or the taxonomy (gtdb). Magmap will merge all these tables and create a new one: each row will correspond to a genome (based on its accno) followed by several columns.

Reviewer: ```suggestion
Magmap accepts several types of metadata as `.csv` files that provides information about the genomes that you will use in the pipeline. Each type of metadata can be used to get different information about you genomes as their quality (CheckM) or the taxonomy (gtdb). Magmap will merge all these tables and create a new one: each row will correspond to a genome (based on its accno) followed by several columns.
```

Reviewer: Actually, we're only accepting taxonomy and genome statistics, so this text gets unnecessarily abstract I think. Or maybe, we don't need this part at all and can keep the header (or change to "Genome metadata input"?) and only describe each particular type below?

Author: I changed the title and part of the text.

> Magmap accepts several types of metadata as `.csv` files that provides information about the genomes that you will use in the pipeline. Each type of metadata can be used to get different information about your genomes. At the moment, Magmap can handle output from CheckM/CheckM2 and gtdb-tk. Magmap will merge all these tables and create a new one: each row will correspond to a genome (based on its accno) followed by several columns.

Reviewer: ```suggestion
Magmap accepts several types of metadata files that provides information about the genomes that you will use in the pipeline. Each type of metadata can be used to get different information about your genomes. At the moment, Magmap can handle output from CheckM/CheckM2 and gtdb-tk as well as standard GTDB metadata files. Magmap will merge all these tables and create a new one: each row will correspond to a genome (based on its accno) followed by several columns.
```

##### gtdb_metadata

> This file contains several information that can be found in gtdb metadata on the official [website](https://data.ace.uq.edu.au/public/gtdb/data/releases/release220/220.0/bac120_metadata_r220.tsv.gz). You can either use directly the same file found there or you can make a custom one.

Reviewer: ```suggestion
With this parameter, you can supply a file like the GTDB metadata files provided on their official [website](https://gtdb.ecogenomic.org/), e.g. [`bac120_metadata_r220.tsv.gz`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release220/220.0/bac120_metadata_r220.tsv.gz). You can either use their files directly or make a custom one. We're using these fields: `A`, `B` etc.
```

Reviewer: @danilodileo : We should fill in the fields we're using so people don't have to replicate all >100 fields in the original files.

Author: done it.

##### gtdb-tk metadata

> This file contains several information that can be obtained as gtdb-tk output, [see](https://ecogenomics.github.io/GTDBTk/files/markers_summary.tsv.html). You can either use directly the output file or you can make a custom one.

Reviewer: ```suggestion
This file should be formatted like the GTDB-Tk output, [see](https://ecogenomics.github.io/GTDBTk/files/summary.tsv.html).
```

##### checkM metadata

> This file contains several information that can be obtained as checkM output, [see](https://github.com/nf-core/test-datasets/blob/magmap/testdata/checkm2.quality_report.tsv). You can either use directly the output file found there or you can make a custom one.

Reviewer: ```suggestion
This file should be formatted like the checkM output, [see](https://github.com/nf-core/test-datasets/blob/magmap/testdata/checkm2.quality_report.tsv).
```

Reviewer: You link to the wrong file: CheckM2, but this is the CheckM section.

Author: I changed the name

> This file should be formatted like the checkM2 output, [see](https://github.com/nf-core/test-datasets/blob/magmap/testdata/checkm2.quality_report.tsv).

Reviewer: This is below the CheckM title but talks about CheckM2. We accept both, don't we?
